index.html

```diff
@@ -0,0 +1,185 @@
+                <ul class="ul-list">
+                    <li>Back End Dev</li>
+                </ul>
+                <ul class="ul-list">
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="paragraph">
+                <h4>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </h4>
+            </div>
+            <div class="btn">
+                <button type="button" class="btn-1">html</button>
+                <button type="button" class="btn-2">css</button>
+                <button type="button" class="btn-3">javaScript</button>
+            </div>
+            <div class="big-btn">
+                <button type="button" class="btn-4">See Project</button>
+            </div>
+        </div>
+
+        <!--End of Second Wrapper ....................-->
+
+        <div class="third-wrapper">
+            <img src="images/Snapshoot-Portfolio-3.png" alt="">
+            <div>
+             <h3>
+                 Tonic
+             </h3>
+         </div>
+         <div class="list-items">
+             <h4>CANOPY</h4>
+             <ul class="ul-list">
+                 <li>Back End Dev</li>
+             </ul>
+             <ul class="ul-list">
+                 <li>2015</li>
+             </ul>
+         </div>
+         <div class="paragraph">
+             <h4>
+                 A daily selection of privately personalized reads; no accounts or sign-ups required.
+             </h4>
+         </div>
+         <div class="btn">
+             <button type="button" class="btn-1">html</button>
+             <button type="button" class="btn-2">css</button>
+             <button type="button" class="btn-3">javaScript</button>
+         </div>
+         <div class="big-btn">
+             <button type="button" class="btn-4">See Project</button>
+         </div>
+     </div>
+
+     <!--End of Third Wrapper...................-->
+
+     <div class="fourth-wrapper">
+        <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        <div>
+         <h3>
+             Multi-Post Stories
+         </h3>
+     </div>
+     <div class="list-items">
+         <h4>CANOPY</h4>
+         <ul class="ul-list">
+             <li>Back End Dev</li>
+         </ul>
+         <ul class="ul-list">
+             <li>2015</li>
+         </ul>
+     </div>
+     <div class="paragraph">
+         <h4>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+         </h4>
+     </div>
+     <div class="btn">
+         <button type="button" class="btn-1">html</button>
+         <button type="button" class="btn-2">css</button>
+         <button type="button" class="btn-3">javaScript</button>
+     </div>
+     <div class="big-btn">
+         <button type="button" class="btn-4">See Project</button>
+     </div>
+ </div>
+<section class="about-me"> 
+    <div>
+        <h1>About Myself</h1>
```

Comment:
- [ ] Kindly use a different heading tag like `h2` because the `h1` has already been used in the introductory part of your portfolio. It is advised to use only one `h1` tag on a page for SEO purposes.